src/components/App/Header/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styles from './styles.module.scss';
```

Comment:
You can change the name of scss file to name of the component. It will produce you better className for debugging.

Reply:
Wow, that's cool!